src/instantiator/characteristic/enumeration-characteristic-instantiator.ts

```diff
@@ -124,4 +130,32 @@ export class EnumerationCharacteristicInstantiator extends CharacteristicInstant
     shouldProcess(nameNode: NamedNode): boolean {
         return this.metaModelElementInstantiator.sammC.EnumerationCharacteristic().equals(nameNode);
     }
+
+    solveBlankNodeValues(resolvedBlankNodes) {
```

Comment:
Change solveBlankNodeValues and add this here:

```
solveBlankNodeValues(resolvedBlankNodes: Array<Quad>) {
        return resolvedBlankNodes.length > 0
            ? resolvedBlankNodes.map(item => this.createLanguageObject(item))
            : '';
    }

    private getPredicateKey(quad: Quad): string {
        return quad.predicate.value.split('#')[1];
    }

    private createLanguageObject(quad: Quad): any {
        return (quad.object as any).language
            ? {value: quad.object.value, language: (quad.object as any).language}
            : quad.object.value;
    }```

Reply:
done